doc/sphinx/source/features.rst

```diff
@@ -10,3 +10,4 @@ Supported OpenCL features, device drivers
    conformance
    extensions
    drivers
+   jpeg_dbk
```

Comment:
I don't think a single DBK warrants a whole section of its own in the doc. Please move this under the section which discusses DBKs/BiKs.

Reply:
Also add a mention of this new (experimental) feature to the next release notes .rst

Reply:
There is no current section to discussing just DBKs/BiKs , only the almaif touches a bit on it. I'll make a new section.